### ISSUE TITLE
Topics Module: Should we maintain the SSP end-points in the module than expecting publishers to pass those?

### ISSUE THREAD

**Issue by pm-harshad-mane** [prebid/Prebid.js#9982]
## Type of issue
Feature request / Question
Refer: https://github.com/prebid/Prebid.js/blob/master/modules/topicsFpdModule.md
```
pbjs.setConfig({
    userSync: {
        ...,
        topics: { 
            maxTopicCaller: 3, // SSP rotation 
            bidders: [{
                bidder: 'pubmatic',
                iframeURL: 'https://ads.pubmatic.com/AdServer/js/topics/topics_frame.html',
                expiry: 7 // Configurable expiry days
            },{
                bidder: 'rubicon',
                iframeURL: 'https://rubicon.com:8080/topics/fpd/topic.html', // dummy URL
                expiry: 7 // Configurable expiry days
            },{
                bidder: 'appnexus',
                iframeURL: 'https://appnexus.com:8080/topics/fpd/topic.html', // dummy URL
                expiry: 7 // Configurable expiry days
            }]
        }
        ....
    }
})
```

The Topics module needs to be configured by publishers with endpoints for each SSP. 
This may take longer to set up and could cause extra effort for publishers.
Also, the endpoints will be the same for all publishers, so It would be easier for SSPs to modify these endpoints when needed if they are maintained directly in the Prebid code.

**Comment by pm-harshad-mane** [comment]
yeah, am expecting the end-points should be maintained in Prebid just like the bidders maintain their user-sync end-points.

**Comment by pm-harshad-mane** [comment]
@patmmccann , the Topics are retrieved by using JS API `document.browsingTopics()`, This API makes sure that callers only receive topics they've observed. 
https://developer.chrome.com/docs/privacy-sandbox/topics/overview/#api-callers-only-receive-topics-theyve-observed
Caller's domain plays an important role here, hence we use iframes. 
I am not clear how Fetch API will be useful here, can you help me understand? Is Chrome providing some other mechanism than `document.browsingTopics()`?

**Comment by patmmccann** [comment]
see #9981 and the accompanying docs ref. It seems like calling document.browsingTopics() is not going to be wise soon instead of using headers on a fetch

**Comment by patmmccann** [comment]
@pm-harshad-mane doesnt this line solve your issue? https://github.com/prebid/Prebid.js/blob/8e3c263c3374040e27fb96073ddd35d5f7b7c334/modules/topicsFpdModule.js#L126
